Makefile

```diff
@@ -130,7 +130,7 @@ CFLAGS          += -G 0 -non_shared -Xcpluscomm -nostdinc -Wab,-r4300_mul
 WARNINGS        := -fullwarn -verbose -woff 624,649,838,712,516,513,596,564,594
 ASFLAGS         := -march=vr4300 -32 -G0
 COMMON_DEFINES  := -D_MIPS_SZLONG=32
-GBI_DEFINES     := -DF3DEX_GBI_2
+GBI_DEFINES     := -DF3DEX_GBI
```

Comment:
Oh, now that you are doing this, could you add `gfx_ucode: f3dex` to the splat yaml file?